db/seeds.rb

```diff
@@ -214,6 +214,12 @@ def random_record_for_org(org, klass)
     email: "recertification_required@example.com",
     status: :recertification_required,
     notes: note.sample
+  },
+  {
+    name: "Pawnee Country Clerk",
```

Comment:
This is really nit-picky -- but I'd like the name to sound more like a 'real' partner.  Say... Pawnee Middle School or Pawnee Junior League or  Pawnee Women's Shelter or  Pawnee Central Library ?

Reply:
Yes, that makes more sense.